cpp/example_code/transcribe/README.md

```diff
@@ -0,0 +1,38 @@
+# Amazon Transcribe code examples for the C++ SDK
+## Overview
+The code examples in this directory demonstrate how to work with the Amazon Transcribe using the AWS SDK for C++.
```

Comment:
the Amazon Transcribe --> Amazon Transcribe

Reply:
fixed